### DISCUSSION TITLE
Netlify deploy UI broken

### DISCUSSION THREAD

**Discussion by tresero** [razonyang/hugo-theme-bootstrap#781] [Q&A]
I looked to see if the cloudflare issue was the same, but it's not, or I don't understand. 

I look at the source code and it seems like the theme isn't importing? I am installing as a hugo module. I think the Netlify docs may be outdated? 

I've never used modules before, but all my other sites deploy with no issues.

[Salsa Blanca Dev](https://salsablanca.netlify.app/)

**Comment by razonyang** [comment]
Hi, that's because the CORS that your site is loading assets from different site. The simplest way to solve it is to change the `baseURL` as `https://salsablanca.netlify.app/` in `config/production/config.toml`.

```
Access to CSS stylesheet at 'https://salsablanca.com/assets/viewer/bundle.min.eb914844636cd41f221f109e99c887bbc3b6b5ffb2af7c664b284cea2d1b54b7.css' from origin 'https://salsablanca.netlify.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
bundle.min.eb914844636cd41f221f109e99c887bbc3b6b5ffb2af7c664b284cea2d1b54b7.css:1          Failed to load resource: net::ERR_FAILED
```

**Reply by tresero** [reply]
Thanks!
